chipy_org/apps/main/views.py

```diff
@@ -21,6 +21,9 @@ def get_context_data(self, **kwargs):
         future_meetings = Meeting.objects.filter(
             when__gt=datetime.datetime.now() - datetime.timedelta(hours=24))
 
+        context["general_sponsors"] = GeneralSponsor.objects.all(
+            ).order_by('?')
```

Comment:
That's Django's notation for random ordering.  For small queries like this, I think this fits the bill (but for much larger queries, it might not be very efficient).